src/form/sections/HeaderFooterMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from '@folio/stripes/components';
+import React from 'react';
+import { FieldArray } from 'react-final-form-arrays';
+import { HeaderFooterTokenType } from '../../types/TokenTypes';
+import HeaderFooterCard from '../../components/Token/HeaderFooter/HeaderFooterCard';
+import { FormattedMessage } from 'react-intl';
+
+export default function HeaderFooterMenu({ name }: { name: string }) {
+  return (
+    <FieldArray name={name}>
+      {({ fields }) => (
+        <>
+          {fields.map((innerName, index) => (
+            <HeaderFooterCard
```

Comment:
same for this component, we render form, but it's card and smth with header/footer